releasey/libs/_github.sh

```diff
@@ -0,0 +1,154 @@
+#!/bin/bash
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+#
+# Git Remote Setup Script
+#
+# This script ensures that the Git repository has the proper remote configuration for Apache Polaris releases:
+# 1. Checks whether there is a Git remote named 'apache' pointing to https://github.com/apache/polaris.git
```

Comment:
I think it's probably better to just work backwards from the assumption "we need the correct target" and not require the other conditions.

1. Checks remotes
2. If one doesn't exist with the correct URI for apache/polaris, error out
3. If it does exist return the name of the remote which will be used